meta/meta.go

```diff
@@ -62,6 +62,7 @@ var (
 	mDBPrefix         = "DB"
 	mTablePrefix      = "Table"
 	mTableIDPrefix    = "TID"
+	mRandomIDPrefix   = "ARID"
```

Comment:
How about `mTableRandomIDPrefix = TARID`, indicate the id was related with table, not a global id.